AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOperation.swift

```diff
@@ -188,7 +206,7 @@ final class InitialSyncOperation: AsynchronousOperation {
             case .failure(let dataStoreError):
                 self.finish(result: .failure(dataStoreError))
             case .success:
-                self.finish(result: .successfulVoid)
+                self.finish(result: .success(self.modelSyncedPayload))
```

Comment:
If you make `modelSyncedPayload` non-optional, you may be able to change the signature of the completion block to take a non-optional modelSyncedPayload--I'm not sure where else you use the callback though.

Reply:
See: https://github.com/aws-amplify/amplify-ios/pull/721#discussion_r477581573